app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/kustomize/BakeKustomizeConfigForm.tsx

```diff
@@ -27,10 +28,10 @@ export class BakeKustomizeConfigForm extends React.Component<
     return (
       <div className="form-horizontal clearfix">
         <div className="container-fluid form-horizontal">
-          <h4>Kustomize File</h4>
+          <h4>Kustomize Options</h4>
           <StageArtifactSelectorDelegate
             artifact={this.getInputArtifact().artifact}
-            excludedArtifactTypePatterns={[]}
+            excludedArtifactTypePatterns={excludeAllTypesExcept(ArtifactTypePatterns.GIT_REPO)}
```

Comment:
So now that we only accept the new git repo artifact type in Kustomize stages, will all previously configured Kustomize stages break?

Reply:
In the UI, yes. We expected this so we called Kustomize support Beta in the last release. We're leaving the old code in place (for now) so it should continue to work. We'll come up with a communication plan for when the old path will be removed.